CHANGELOG.md

```diff
@@ -4,7 +4,17 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.6.0 (2022-02-16)
+This release adds major new features since the 2.5.2 release, among
+them:
+
+* Compression in continuous aggregates
+* Experimental support for timezones in continuous aggregates
```

Comment:
Please add "Experimental support for monthly buckets in continuous aggregates". This was not available in 2.5.x https://docs.timescale.com/api/latest/hyperfunctions/time_bucket_ng/

Reply:
```suggestion
* Experimental support for timezones in continuous aggregates
* Experimental support for monthly buckets in continuous aggregates
```